script/main.js

```diff
@@ -146,6 +145,27 @@ function fetchNote(){
 
 };
 
+async function createNote(){
+    dataMode = false;
```

Comment:
try to use "try catch" for all requests and make sure we have good error handling that sends feedback to the user. other wise the user never knows when it went wrong.